openstack/utils/templates/_proxysql.tpl

```diff
@@ -34,7 +34,7 @@
 {{- end }}
 
 {{- define "utils.proxysql.volumes" }}
-  {{- if .Values.proxysql }}
+  {{- if .Values.proxysql -}}
```

Comment:
Is this change necessary? I would have assumed the whitespace is consumed by the next `if` anyways.

Reply:
Ah, that one not... I've removed it